projects/kit/components/message/message.options.ts

```diff
@@ -0,0 +1,15 @@
+import type {Provider} from '@angular/core';
+import {tuiCreateToken, tuiProvideOptions} from '@taiga-ui/cdk/utils/miscellaneous';
+import type {TuiAppearanceOptions} from '@taiga-ui/core/directives/appearance';
+
+export const TUI_MESSAGE_DEFAULT_OPTIONS: TuiAppearanceOptions = {
+    appearance: 'neutral',
+};
+
+export const TUI_MESSAGE_OPTIONS = tuiCreateToken(TUI_MESSAGE_DEFAULT_OPTIONS);
```

Comment:
Let's not introduce options for now, if we only have appearance. Just provide it directly.

Reply:
Removed, provided